apidump/apidump.go

```diff
@@ -76,6 +76,61 @@ type Args struct {
 	Plugins []plugin.AkitaPlugin
 }
 
+func DumpPacketCounters(interfaces map[string]interfaceInfo, inboundSummary *trace.PacketCountSummary, outboundSummary *trace.PacketCountSummary) {
+	// Using a map gives inconsistent order when iterating (even on the same run!)
+	directions := []kgxapi.NetworkDirection{kgxapi.Inbound, kgxapi.Outbound}
+	toReport := []*trace.PacketCountSummary{inboundSummary}
+	if outboundSummary != nil {
+		toReport = append(toReport, outboundSummary)
+	}
+
+	// Debugging info: packets per interface and prot
+	printer.Stderr.Debugf("==================================\n")
+	printer.Stderr.Debugf("Packets per interface:\n")
+	printer.Stderr.Debugf("%15v %8v %7v %5v %5v %5v\n", "interface", "dir", "packets", "req", "resp", "unk")
```

Comment:
Why not write out "requests", "responses", and "unknown" in full? Similarly elsewhere.

Reply:
🤷 I find it easier to read

```
[DEBUG]       interface      dir packets   req  resp   unk
[DEBUG]              lo  INBOUND       0     0     0     0
[DEBUG]              lo OUTBOUND       0     0     0     4
```

than
```
[DEBUG]       interface      dir packets requests responses unknown
[DEBUG]              lo  INBOUND       0        0         0       0
[DEBUG]              lo OUTBOUND       0        0         0       4
```

since I thought 99999 was a reasonable (but non-enforced) upper bound on the number of packets.  I thought about going down to 4 characters per column :)